src/elements/GridItem.vue

```diff
@@ -25,11 +25,19 @@ export default {
     },
     /**
      * Sets the size of the column. Prefix with `sm-` or `lg-`. Based on a 12 column grid.
+     * The inclusion of `auto` will set that grid item to have a width based on the width and height of the content.
      */
     columns: {
       type: String,
       default: "",
     },
+    /**
+     * Sets the vertical alignment of the item. `start`, `center`, or `end`. For horizontal alignment, please look at the wrapper component.
```

Comment:
In the example, you use `v-start`, so maybe we should make sure the comments are consistent.

Reply:
I updated this to be consistent.